cnd/src/config.rs

```diff
@@ -23,6 +23,7 @@ pub struct Data {
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub struct Network {
     pub listen: Vec<Multiaddr>,
+    pub use_tor: bool,
```

Comment:
At least at the moment, this is redundant. You could detect that from the Multiaddress that is configured.

However, if we were to give `cnd` more control over this by configuring Tor itself by the use of the cookie file or something, then we'd probably need a configuration option. Actually, the presence of the cookie file could be the enabling bit :)

Reply:
Good thinking re asserting use of tor based on the address. Will add.

Reply:
Done as suggested.